loadtest/control/simulcontroller/actions.go

```diff
@@ -881,10 +881,15 @@ func searchChannels(u user.User) control.UserActionResponse {
 		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
 
-	// We simulate the user typing up to 4 characters when searching for
-	// a channel. This is an arbitrary value which fits well with the current
+	// numChars simulates how many characters does a user type
+	// to search for a channel. This is an arbitrary value which fits well with the current
 	// frequency value for this action.
-	return control.EmulateUserTyping(channel.Name[:1+rand.Intn(4)], func(term string) control.UserActionResponse {
+	numChars := 4
+	if len(channel.Name) < numChars {
+		return control.UserActionResponse{Info: "channel name too short"}
```

Comment:
Shouldn't we continue to search for the channel even if it's name is 2 letter? As far as I recall the idea for using `4` characters is that the user may stop at some point since the results will narrow down to a reasonable number so they can pick one of them.

Reply:
Oh you mean have a max limit of `numChars` to be 4, but if the channel name is smaller, use that?

Reply:
Yep.